tests/unit/s2n_resume_test.c

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#include "s2n_test.h"
+
+#include "tls/s2n_tls.h"
+#include "utils/s2n_safety.h"
+#include "tls/s2n_resume.h"
+
+int main(int argc, char **argv)
+{
+    BEGIN_TEST();
+
+    /* s2n_config_set_initial_ticket_count */
```

Comment:
Add a unit test for the upper limit for the initial ticket count here (256 tickets).

Reply:
What new information would this give us? We already know that a uint8_t can fit in a uint16_t variable.